pipeline/docker/__init__.py

```diff
@@ -14,9 +15,10 @@ def create_pipeline_api(
     **environment_variables,
 ):
     paths = []
+    Path(output_dir).mkdir(parents=True, exist_ok=True)
 
     for pipeline_graph in pipeline_graphs:
-        graph_path = pipeline_graph.name + ".graph"
+        graph_path = os.path.join(output_dir, pipeline_graph.name + ".graph")
```

Comment:
nbd but I always thought `pathlib.Path`'s overload of the `/` operator was kinda cute 😄 So we could do

```python
p = Path(output_dir)
p.mkdir(parents=True, exist_ok=True)
# …
graph_path = output_dir / pipeline_graph.name + ".graph"
```

Reply:
I feel it can be obscure to the unaware dev, if you have strong feelings about it I can change though

Reply:
Nah not strong.